scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/struct/ClassParameterMemberParser.java

```diff
@@ -23,25 +26,50 @@
 
 		final ArrayList<SynthesizedParameterMember<?>> items = new ArrayList<>();
 		final ArrayList<ValidityProblem> problems = new ArrayList<>();
-		final Set<String> names = new HashSet<>();
 
 		// NB: Reject abstract classes.
 		org.scijava.struct.Structs.checkModifiers(source.getName() + ": ", problems, source.getModifiers(), true, Modifier.ABSTRACT);
 
-		// obtain a parameterData (preferably one that scrapes the javadoc)
-		ParameterData paramData;
+		// Obtain source's Op method.
+		Method opMethod;
 		try {
-			paramData = new JavadocParameterData(source);
-		} catch(NullPointerException | IllegalArgumentException e) {
-			paramData = new SynthesizedParameterData();
+			opMethod = getOpMethod(source);
 		}
+		catch (NoSuchMethodException e) {
+			problems.add(new ValidityProblem("OpClass " + source +

```

Comment:
`OpClass " + source` &rarr; `Class " + source.getName()`

Reply:
Fixed in 6e807112